external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDataConsumerSuite.scala

```diff
@@ -95,10 +94,13 @@ class KafkaDataConsumerSuite extends SharedSQLContext with PrivateMethodTester {
       try {
         val range = consumer.getAvailableOffsetRange()
         val rcvd = range.earliest until range.latest map { offset =>
-          val bytes = consumer.get(offset, Long.MaxValue, 10000, failOnDataLoss = false).value()
-          new String(bytes)
+          val record = consumer.get(offset, Long.MaxValue, 10000, failOnDataLoss = false)
+          (
+            new String(record.value()),
```

Comment:
Needs a character encoding. I'd also find a way to not split the tuple this way, for clarity that it's a tuple.

Reply:
Fixed. I found that some other constructor calls also omitting the character encoding. They seem to be cleaned up later.